src/pages/msme/ExternalOnboardingCompleted.tsx

```diff
@@ -0,0 +1,46 @@
+import { useT } from '@transifex/react';
```

Comment:
How do borrowers reach this page? I don't know what the flow is meant to be.

Reply:
From the reminder email `EXTERNAL_ONBOARDING_DONE_URL`